src/content/docs/en/reference/experimental-flags/sessions.mdx

```diff
@@ -72,7 +72,8 @@ Configure `session.driver` with the unstorage `driver` name that corresponds to
 Some drivers may need extra packages to be installed. For example, Upstash requires the `@upstash/redis` package. Some drivers may also require environment variables or credentials to be set.
 
 <ReadMore>
-  See [the unstorage driver docs](https://unstorage.unjs.io/drivers) for more information about each driver, including its `driver` name, any additional dependencies, and usage.
+  See [the unstorage driver docs](https://unstorage.unjs.io/drivers) for more information about each
+  driver, including its `driver` name, any additional dependencies, and usage.
```

Comment:
```suggestion
  See [the unstorage driver docs](https://unstorage.unjs.io/drivers) for more information about each driver, including its `driver` name, any additional dependencies, and usage.
```
Suggest reverting this back as I'm not sure it was intentional? It doesn't make a visual change, and is one less thing for all the translators to change.

Reply:
Yeah, that was the formatter